src/Tempest/Database/Migrations/CreateMigrationsTable.php

```diff
@@ -15,14 +16,14 @@ public function getName(): string
         return '0000-00-00_create_migrations_table';
     }
 
-    public function up(): CreateTableStatement|null
+    public function up(): QueryStatement|null
```

Comment:
changed the return type to indicate we're doing a query. Doesn't matter what kind